packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -16,7 +16,6 @@ contract L1StandardBridge_Getter_Test is Bridge_Initializer {
         assert(L1Bridge.OTHER_BRIDGE() == L2Bridge);
         assert(L1Bridge.messenger() == L1Messenger);
         assert(L1Bridge.MESSENGER() == L1Messenger);
-        assertEq(L1Bridge.version(), "1.0.0");
```

Comment:
We should probably just change this to `1.1.0`.

```suggestion
        assertEq(L1Bridge.version(), "1.1.0");
```

Reply:
I deleted this because none of the other contract tests test for the version

Reply:
I think we should follow up with a standard pattern for testing the semver

Reply:
ok i added it back, but i still think we should come up with a unified plan for testing semver values